pkg/types/baremetal/platform.go

```diff
@@ -238,4 +238,9 @@ type Platform struct {
 	// +kubebuilder:validation:Format=ip
 	// +optional
 	BootstrapExternalStaticDNS string `json:"bootstrapExternalStaticDNS,omitempty"`
+
+	// DisableVirtualMediaTLS enables or disables TLS when virtual media is
+	// used to boot a node with virtual media images.
+	// +optional
+	DisableVirtualMediaTLS bool `json:"disableVirtualMediaTLS,omitempty"`
```

Comment:
This is not wrong. But first, adding any fields require an agreement from the installer team. Second, doing so will make your change not backportable.

Reply:
No objection to adding this field from the installer side, but it would indeed be an obstacle to backporting.